test/custom_number_test.dart

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Perhaps rename this file to something a bit more expressive? `custom_number_test` could be anything.

Reply:
It looks consistent with the content of the test that uses a `MyNum` class.

If you have better name in mind, I'm open to suggestions.

Reply:
Maybe just `customnumber_test.dart`? I first read it as custom numbertest, which confused me. But I am just bikeshedding right now I think.

Reply:
(You can leave it as-is, I don't know a better name either.)